demo/src/app/app.component.ts

```diff
@@ -1,32 +1,33 @@
-import { ViewportScroller } from '@angular/common';
-import { Component, NgZone, OnInit } from '@angular/core';
-import { ActivatedRoute } from '@angular/router';
-import { filter } from 'rxjs/operators';
+import {ViewportScroller} from '@angular/common';
+import {HttpClient} from '@angular/common/http';
+import {Component, NgZone, OnInit} from '@angular/core';
+import {ActivatedRoute} from '@angular/router';
+import {filter, pluck} from 'rxjs/operators';
 
-import { componentsList } from './shared';
-import { Analytics } from './shared/analytics/analytics';
+import {environment} from '../environments/environment';
+import {componentsList} from './shared';
+import {Analytics} from './shared/analytics/analytics';
 
-@Component({
-  selector: 'ngbd-app',
-  templateUrl: './app.component.html'
-})
+
+@Component({selector: 'ngbd-app', templateUrl: './app.component.html'})
 export class AppComponent implements OnInit {
+  downloadCount = '---,---';
   navbarCollapsed = true;
 
   components = componentsList;
 
   constructor(
-    private _analytics: Analytics,
-    route: ActivatedRoute,
-    vps: ViewportScroller,
-    zone: NgZone
-  ) {
-    route.fragment
-      .pipe(filter(fragment => !!fragment))
-      .subscribe(fragment => zone.runOutsideAngular(() => requestAnimationFrame(() => vps.scrollToAnchor(fragment))));
-  }
+      private _analytics: Analytics, route: ActivatedRoute, vps: ViewportScroller, zone: NgZone,
+      httpClient: HttpClient) {
+    route.fragment.pipe(filter(fragment => !!fragment))
+        .subscribe(fragment => zone.runOutsideAngular(() => requestAnimationFrame(() => vps.scrollToAnchor(fragment))));
 
-  ngOnInit(): void {
-    this._analytics.trackPageViews();
+    if (environment.production) {
+      httpClient.get('https://api.npmjs.org/downloads/point/last-month/@ng-bootstrap/ng-bootstrap')
```

Comment:
maybe add pipe → `catchError(() => of(''))`or the error handler in subscribe → `() => this.downloadCount = ''`

So it would:
- not shout in the console in case of issues
- display just the `npm` logo without dashes

![Screen Shot 2020-01-27 at 11 31 26](https://user-images.githubusercontent.com/8074436/73167682-92f7c580-40f8-11ea-810f-84cbbcdcd353.png)

Reply:
done